lib/internal/bootstrap/node.js

```diff
@@ -395,7 +395,7 @@
     }
 
     function makeSetter(name) {
-      return util.deprecate(function(value) {
+      return util.deprecate((value) => {
         Object.defineProperty(this, name, {
```

Comment:
`this` is used here so that we should not change here, should we?

Reply:
sorry, missed that part. I'll change that part back to function as well